modules/postgresql/read_replica.tf

```diff
@@ -32,7 +32,7 @@ locals {
 resource "google_sql_database_instance" "replicas" {
   count                 = "${var.read_replica_size}"
   project               = "${var.project_id}"
-  name                  = "${var.name}-replica${count.index}"
+  name                  = "${var.name}-replica${var.read_replica_name_suffix}${count.index}"
```

Comment:
I would like to put `read_replica_name_suffix` before `count.index`. Thoughts?

Reply:
Good pickup - that would be another place where this issue would present itself. Ordering also looks good - it would be odd the other way around with a numeral in the middle of the read replica name

Reply:
Agreed.